bin/symfony-repl

```diff
@@ -33,4 +33,6 @@ $container = $kernel->getContainer();
 $doctrine = $container->get('doctrine');
 $container->get('cache_clearer')->clear($kernel->getCacheDir());
 
-extract(Psy\Shell::debug(compact('kernel', 'container', 'doctrine')));
+$sh = new \Psy\Shell();
+$sh->setScopeVariables(compact('kernel', 'container', 'doctrine'));
```

Comment:
While Doctrine is optional in Symfony, it is by far the most popular ORM\* and it is the one featured in the Symfony documentation. Though, to be compatible with ORM-less/non-Doctrine users, we could do the following:

```php
if ($container->has('doctrine')) {
    $doctrine = $container->get('doctrine');
}
```

As far as accessing it instead through the container goes: I find my ORM to be the component I use the most within the REPL, so having the variable there does save some time.

How do you think?

\* - See [Doctrine](https://packagist.org/packages/doctrine/orm), [RedBean](https://packagist.org/packages/gabordemooij/redbean), and [Propel](https://packagist.org/packages/propel/propel).